### DISCUSSION TITLE
hide unavailable videos

### DISCUSSION THREAD

**Discussion by lamyergeier** [stashapp/stash#4716] [Ideas]
I store Videos on my laptop and on removable Harddrive. How should I manage this 2 set of data? Should I scan everything together? 

The videos in laptop will always be available but the videos in external  Harddrive will be plugged in rarely and will be available only when the Harddrive is plugged in.

I don't want the scenes from Harddrive to show up when it's not connected.

Can stash hide unavailable videos?

**Comment by DogmaDragon** [comment]
You can filter them out. With path filter, you can set it to exclude scenes that are on external drive and set that filter as default. You would just need to remove the filter to see them again.
